File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    db = {d[0] : d[1] for d in db}
    
    if id_pw[0] in db:
        if id_pw[1] == db[id_pw[0]]:
            return "login"
        else:
            return "wrong pw"
    else:
        return "fail"

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.js
function solution(id_pw, db) {
    let dict = new Map(db)
    
    if (dict.has(id_pw[0])) {
        return dict.get(id_pw[0]) == id_pw[1] ? "login" : "wrong pw"
    } else {
        return "fail"
    }
}